client/server/bundler/sections-loader.js

```diff
@@ -55,12 +55,29 @@ function printSectionsAndPaths( sections ) {
 	}
 }
 
+function filterSections( sections ) {
```

Comment:
Given that this is a no-op in all other environments, should we just explicitly name it `filterSectionsInDevelopment`?  The name seems overly general currently.